pkg/scheduler/framework/runtime/framework.go

```diff
@@ -32,9 +32,9 @@ import (
 	"k8s.io/client-go/tools/events"
 	"k8s.io/component-helpers/scheduling/corev1"
 	"k8s.io/klog/v2"
-	"k8s.io/kube-scheduler/config/v1beta1"
 	"k8s.io/kubernetes/pkg/scheduler/apis/config"
 	"k8s.io/kubernetes/pkg/scheduler/apis/config/scheme"
+	"k8s.io/kubernetes/pkg/scheduler/apis/config/v1beta2"
```

Comment:
I think we need still depend on `k8s.io/kube-scheduler/config/v1beta2`

Reply:
oops, you are correct, thanks for spotting

Reply:
This has been updated @alculquicondor